expo/gbe/views/review_volunteer_view.py

```diff
@@ -1,103 +1,44 @@
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import (
-    get_object_or_404,
-    render,
-)
-from django.http import HttpResponseRedirect
-from django.core.urlresolvers import reverse
-from expo.gbe_logging import log_func
-from gbe.functions import (
-    validate_perms,
-    get_conf,
-)
-from gbe.models import (
-    BidEvaluation,
-    Conference,
-    Volunteer,
-)
+from gbe.models import Volunteer
+
 from gbe.forms import (
     BidEvaluationForm,
-    BidStateChangeForm,
+    VolunteerBidStateChangeForm,
```

Comment:
Please leave it as it was.  I removed the VolunteerBidStateChangeForm from this class intentionally last year, when I created the assign volunteer view.  I was actually trying to remove the form entirely in a separate branch connected to https://github.com/bethlakshmi/GBE2/issues/180

It's not a great design and I think it can be removed with a simpler/better view.  There's not reason to include it here.